src/context/ThemeProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect, ReactNode } from 'react';
+import { ThemeContext, Theme } from './theme';
+
+interface ThemeProviderProps {
+  children: ReactNode;
+}
+
+export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
```

Comment:
```suggestion
export const ThemeProvider  = ({ children }: ThemeProviderProps) => {
```
чаще так используют сейчас